config/initializers/01_redis.rb

```diff
@@ -0,0 +1,3 @@
+REDIS_POOL = ConnectionPool.new(size: IdentityConfig.store.redis_pool_size) do
```

Comment:
what do we use redis for in this app? do we need it?

Reply:
i think we will need it later so i added that.

Reply:
I think we don't have any use cases defined yet, so I'd opt to remove it for now, it's very easy to add back in later if needed

Reply:
sure!